core/controllers/outgoing_emails.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Controllers for sending outgoing emails."""
+
+from core.controllers import base
+from core.domains import email_manager
+import json
+
+
+def write_email_contents(user_organization, user_comment):
+    """Compose email body with given user info.
+
+    Arguments:
+        user_organization: str. User's organization name to be written at the
+            top of the email body.
+        user_comment: str. User's comment to be used as the primary email
+            content.
+
+    Returns:
+        str. Email body to be sent.
+    """
+
+    email_contents = ('Organization: %s\n' % user_organization)
+    email_contents += user_comment
+    return email_contents
+
+
+def write_email_subject(user_email_address):
```

Comment:
Not sure this is right. If we want it to be directly replyable, then the subject should be something like "Partnering with Oppia" or similar depending on which page it comes from.

Reply:
Oh wow, I forgot that this same form exists on 2 diff pages.

So if it came from the "Partnerships" page, the title should be something like:
"Oppia Foundation Website - Partnering with Oppia from [user_email]"

When the "Volunteer" page form get filled out, the title should then be:
"Oppia Foundation Website - Volunteering at Oppia from [user_email]"

Does this sound good?

Reply:
Let's go with just "Partnering with Oppia" and "Volunteering with Oppia". Then the replies will be "Re: Partnering with Oppia" etc. which seems more natural.

Reply:
Done!